sklearn/feature_selection/_rfe.py

```diff
@@ -624,6 +628,20 @@ class RFECV(RFE):
     array([1, 1, 1, 1, 1, 6, 4, 3, 2, 5])
     """
 
+    _parameter_constraints = {
+        **RFE._parameter_constraints,  # type: ignore
+        "min_features_to_select": [Interval(Integral, 0, None, closed="neither")],
+        "cv": [
+            None,
+            Interval(Integral, 0, None, closed="neither"),
```

Comment:
Those two should already be included in `cv_object`.

Reply:
Removed.